src/sparsevector.jl

```diff
@@ -135,6 +141,12 @@ function nonzeros(x::SparseColumnView)
 end
 nonzeros(x::SparseVectorView) = nonzeros(parent(x))
 
+function nonzeros(x::SparseVectorPartialView)
+    (first_idx, last_idx) = _partialview_end_indices(x)
+    nzvals = nonzeros(parent(x))
+    return @view(nzvals[first_idx:last_idx])
```

Comment:
I vaguely remember people saying that you should use the view function directly.

Reply:
done, might be linked to compile time?